tests/test_keras_integration.py

```diff
@@ -440,62 +478,66 @@ def test_override_tensorboard(self):
             model.build((None, 2))
             push_to_hub_keras(
                 model,
-                repo_path_or_name=f"{WORKING_REPO_DIR}/{REPO_NAME}",
+                repo_id=repo_id,
                 log_dir=f"{tmpdirname}/tb_log_dir",
                 api_endpoint=ENDPOINT_STAGING,
-                use_auth_token=self._token,
-                git_user="ci",
-                git_email="ci@dummy.com",
+                token=self._token,
             )
+
             os.makedirs(f"{tmpdirname}/tb_log_dir2")
             with open(f"{tmpdirname}/tb_log_dir2/override.txt", "w") as fp:
                 fp.write("Keras FTW")
             push_to_hub_keras(
                 model,
-                repo_path_or_name=f"{WORKING_REPO_DIR}/{REPO_NAME}",
+                repo_id=repo_id,
                 log_dir=f"{tmpdirname}/tb_log_dir2",
                 api_endpoint=ENDPOINT_STAGING,
-                use_auth_token=self._token,
-                git_user="ci",
-                git_email="ci@dummy.com",
+                token=self._token,
             )
 
-            model_info = HfApi(endpoint=ENDPOINT_STAGING).model_info(
-                f"{USER}/{REPO_NAME}",
-            )
+            model_info = HfApi(endpoint=ENDPOINT_STAGING).model_info(repo_id)
+            # TODO: this test fails. Both tensorboard.txt and override.txt are in the
+            #       repo at the end.
             self.assertTrue(
                 "logs/override.txt" in [f.rfilename for f in model_info.siblings]
             )
             self.assertFalse(
                 "logs/tensorboard.txt" in [f.rfilename for f in model_info.siblings]
             )
```

Comment:
Related to my previous message about deleting the logs on the hub while pushing new logs.